inspectit-oce-core/build.gradle

```diff
@@ -43,9 +42,21 @@ dependencies {
     implementation(
             'org.springframework.boot:spring-boot:2.1.1.RELEASE',
             'org.yaml:snakeyaml:1.23',
+
+            //data validation
             'javax.validation:validation-api:2.0.0.Final',
```

Comment:
this one is not included as part of `org.hibernate.validator:hibernate-validator:6.0.2.Final`